packages/@react-spectrum/breadcrumbs/docs/Breadcrumbs.mdx

```diff
@@ -0,0 +1,198 @@
+  return (
+    <Breadcrumbs>
+      <ExampleItem uniqueKey="Folder 1">Folder 1</ExampleItem>
+      <ExampleItem uniqueKey="Folder 2">Folder 2</ExampleItem>
+      <ExampleItem uniqueKey="Folder 3">Folder 3</ExampleItem>
+    </Breadcrumbs>
+  );
+}
+```
+
+### Internationalization
+
+In order to internationalize Breadcrumbs, the strings of all child items should be localized.
+
+## Events
+
+Use the `onAction` prop as a callback to handle press events on items.
+The `uniqueKey` prop from the selected item will be passed into the callback.
+
+```tsx example
+function Example() {
+  let folders = [
+    {id: 1, label: 'Folder 1'},
+    {id: 2, label: 'Folder 2'},
+    {id: 3, label: 'Folder 3'}
+  ];
+  let [folderId, setFolderId] = React.useState();
+  return (
+    <div>
+      <Breadcrumbs onAction={(a) => setFolderId(a)}>
+        {folders.map(f => <Item key={f.id} uniqueKey={f.id}>{f.label}</Item>)}
+      </Breadcrumbs>
+      <p>Last Selected Folder: {folderId}</p>
+    </div>
+  );
+}
+```
+
+## Props
+
+<PropTable component={docs.exports.Breadcrumbs} links={docs.links} />
+
+## Visual Options
+
+### Size
+
+The small size is useful to maintain Breadcrumbs while reducing the height required to give users context.
+
+```tsx example
+<Breadcrumbs size="S">
+  <Item uniqueKey="Folder 1">Folder 1</Item>
+  <Item uniqueKey="Folder 2">Folder 2</Item>
+</Breadcrumbs>
+```
+
+The medium size is the default.
+```tsx example
+<Breadcrumbs size="M">
+  <Item uniqueKey="Folder 1">Folder 1</Item>
+  <Item uniqueKey="Folder 2">Folder 2</Item>
+</Breadcrumbs>
+```
+
+The large size places emphasis on the selected item as a page title or heading.
+
+```tsx example
+<Breadcrumbs size="L">
+  <Item uniqueKey="Folder 1">Folder 1</Item>
+  <Item uniqueKey="Folder 2">Folder 2</Item>
+</Breadcrumbs>
+```
+
+### Heading
+
+Use the `isHeading` prop to add a role of "heading" to the highlighted item.
+If a change to the aria-level is required, use the `headingAriaLevel` prop.
+
+```tsx example
+<Breadcrumbs isHeading headingAriaLevel={2}>
+  <Item uniqueKey="Folder 1">Folder 1</Item>
+  <Item uniqueKey="Folder 2">Folder 2</Item>
+  <Item uniqueKey="Folder 3">Folder 3</Item>
+</Breadcrumbs>
+```
+
+### Visible Items
+
+Breadcrumbs truncate by default when there is not enough room to display all items.
+
+Truncation of breadcrumb items begins when either there is not enough room to display all items,
+or if there are 5 or more breadcrumbs to display.
+
+In order to manually control the truncation, use the `maxVisibleItems` prop.
+
+```tsx example
+<Breadcrumbs maxVisibleItems={2}>
+  <Item uniqueKey="Folder 1">Folder 1</Item>
+  <Item uniqueKey="Folder 2">Folder 2</Item>
+  <Item uniqueKey="Folder 3">Folder 3</Item>
+</Breadcrumbs>
+```
+
+### Root Context
+
+Some applications find that displaying the root item is useful for orienting users.
+This variation keeps the root visible when other items are truncated into the menu.
+
+```tsx example
+<Flex width="300px">
```

Comment:
Breadcrumbs `width` for this example is ~258px, and its not auto collapsed... maybe to do something like:
```
<Flex width="200px">
  <Breadcrumbs maxVisibleItems="auto" showRoot>
  ...
```

I think ^ will show 3 items: root + icon + current

Reply:
<img width="480" alt="Screen Shot 2020-03-26 at 4 14 49 PM" src="https://user-images.githubusercontent.com/1123335/77705454-f5483780-6f7c-11ea-9df4-9b25e64cad68.png">
@pst67662 i may be doing something wrong -- but it seems to overflow the container when using width `200px` and `maxVisibleItems="auto"`